tensorflow/lite/ios/BUILD.apple

```diff
@@ -34,6 +35,17 @@ sh_binary(
     ],
 )
 
+sh_binary(
```

Comment:
Instead of adding a separate file for hiding symbols in xcframeworks, we can reuse the existing script hide_symbols_with_allowlist.sh.

Actually I have a pending CL that implements this when we added support for xcframework. I abandoned it because the xcframeworks generated using rule apple_static_xcframework aren't compatible with the structure expected by CocoaPods. So we end up building the frameworks for each arch and generate the xcframework bundle afterwards.

If you really need this I can revive my CL.

Reply:
> Instead of adding a separate file for hiding symbols in xcframeworks, we can reuse the existing script hide_symbols_with_allowlist.sh.

I started with that but it's pretty significantly different, I also assumed it would be short lived. I can replace that script if you think it would be ok for us to drop the `ios_static_framework` targets entirely now? But if you have a working combined version that would be fine here too!



> I abandoned it because the xcframeworks generated using rule apple_static_xcframework aren't compatible with the structure expected by CocoaPods. So we end up building the frameworks for each arch and generate the xcframework bundle afterwards.

Like I mentioned above the public `rules_apple` (which I am a maintainer of) support xcframeworks bundling static frameworks, so what I produce with this PR looks exactly like the binary you linked above.